.github/workflows/docker_build.yml

```diff
@@ -1,44 +1,32 @@
-name: Docker Image Build and Push (Debug)
+  name: Docker Image Build and Push
 
-on:
-  push:
-    branches:
-      - '*'
-  pull_request:
-    branches:
-      - main
-    types: [opened, synchronize, reopened, ready_for_review, review_requested]
-  schedule:
-    - cron: '0 0 * * *'
+  on:
+    push:
+      branches:
+        - main
 
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Login to Docker Hub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      
-      - name: Debug
```

Comment:
Why did you remove the debug step?

Reply:
Because it isn't required for the main branch - I just needed it during the development